SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -86,7 +86,7 @@ Plot {
 		})
 	}
 	plotColor_ { |c|
-		if(c.isArray.not) { c = c.bubble };
+		colors = colors.as(Array);
```

Comment:
should't this be `c = c.as(Array)`?

Reply:
The whole method can just be:
```
plotColor_ { |c|
		plotColor = c.as(Array);
}

Reply:
yes, of course – thank you.

Reply:
Thanks! And sorry, I should have been more clear about the differences in `Plot:plotColor` and `Plotter:plotColor`.